draft-ietf-httpbis-digest-headers.md

```diff
@@ -281,6 +281,7 @@ A recipient MAY ignore any or all of the representation-data-digests in a Digest
 field. This allows the recipient to choose which digest-algorithm(s) to use for
 validation instead of verifying every received representation-data-digest.
 
+An intermediary MUST NOT alter the value of a received `Digest` field.
```

Comment:
Do we need this? Isn't it just restating a requirement about end-to-end headers? 

Perhaps we can jsut reference to semantics here too.

Reply:
The "considerations on new fields" in Semantics, asks to explicit whether intermediaries can  alter the field and how.

Reply:
```suggestion
```
No normative language. Discussed with @martinthomson and @mnot at interim 2020-10